.babelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "presets": [
+    "react",
+    "es2015"
```

Comment:
I would actually favor `env` over `es2015`. ES2015 is already two years old, there have been additions to the language since then.